.travis.yml

```diff
@@ -3,19 +3,8 @@ mono: none
 dotnet: 2.1.102
 
 script:
-  - dotnet run -p generator/src/generator.csproj -- . "https://fuse-open.github.io/docs/"
-  - |
-    cp -r generated _site
-    cp -r media _site/
-    cp -r assets/fonts/. _site/
-    cp -r assets/font-awesome-4.7.0/fonts/. _site/
-    cp assets/font-awesome-4.7.0/css/font-awesome.min.css _site/font-awesome.css
-    cp assets/bootstrap-4.0.0/css/bootstrap.min.css _site/bootstrap.css
-    cp assets/jquery-3.3.1/jquery-3.3.1.min.js _site/jquery.js
-    cp assets/popper.js-1.14.1/dist/umd/popper.min.js _site/popper.js
-    cp assets/bootstrap-4.0.0/js/bootstrap.min.js _site/bootstrap.js
-    cp assets/prism/prism.{css,js} _site/
-    cp assets/site.{css,js} _site/
+  - dotnet run -p generator/src/generator.csproj -- . "https://fuse-open.github.io/docs/" _site
```

Comment:
Should the URL be updated to https://fuseopen.com/docs/ now?

Reply:
@mortend: Yes, it should, good point. I've added a commit for that!